frontend/src/app/pipes/absence-status-pipe.ts

```diff
@@ -0,0 +1,21 @@
+import { Pipe, PipeTransform } from "@angular/core";
+
```

Comment:
Az Absence.model.ts-be megvannak ezek az enumok onnan kellene őket kihúzni meg rendberakni mert szerintem nem jók